hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheProxy.java

```diff
@@ -153,7 +154,7 @@ protected AbstractCacheProxy(CacheConfig cacheConfig, NodeEngine nodeEngine, ICa
 
     @Override
     public V get(K key, ExpiryPolicy expiryPolicy) {
-        final Future<V> f = getAsync(key, expiryPolicy);
+        final InternalCompletableFuture<V> f = getAsync(key, expiryPolicy);
```

Comment:
But, in here we just use `f.get()`. So there is no need to explicit `InternalCompletableFuture`.